inc/saihash.h

```diff
@@ -39,89 +39,216 @@
 typedef enum _sai_native_hash_field_t
 {
     /**
-     * @brief Native hash field source IP.
+     * @brief Native hash field source IPv4.
      *
-     * Also, refers to the outer source IP
+     * Also, refers to the outer source IPv4
      * in case for encapsulated packets
      */
-    SAI_NATIVE_HASH_FIELD_SRC_IP = 0,
+    SAI_NATIVE_HASH_FIELD_SRC_IPV4 = 0,
```

Comment:
it looks like there is no significance for adding the enum number. it is copied from original header. I think we can remove the enum number in this pr.

https://github.com/opencomputeproject/SAI/commit/6075277f5aca38478b919b7fa786db9b4c082eb0#diff-048b4a2106e6b4a072d5c388e078c273

Reply:
Done